pkg/cloud/azure/actuators/machine/actuator_test.go

```diff
@@ -658,7 +658,10 @@ func TestMachineEvents(t *testing.T) {
 
 	for _, tc := range cases {
 		t.Run(tc.name, func(t *testing.T) {
-			cs := controllerfake.NewFakeClientWithScheme(scheme.Scheme, tc.credSecret, infra)
+			cs := controllerfake.NewClientBuilder().WithScheme(scheme.Scheme).
```

Comment:
to double check for review. see commit comment at 710be503b0d68e9a3ae33a08e0f81fde0408bada